R/utils.R

```diff
@@ -598,3 +598,49 @@ wait_for_nonmem.list <- function(.mod, .time_limit = 300, .interval = 5) {
 
 }
 
+
+#' Replace BBI_NULL_NUM and BBI_NULL_STR with NA_real_ or NA_character_
+#'
+#' @param .vals a numeric or character vector
+#'
+#' @importFrom purrr map
+#' @keywords internal
+set_bbi_null <- function(.vals){
+  map(.vals, function(.val){
+    if(inherits(.val, "character") && .val == BBI_NULL_STR && !is.na(.val)){
+      return(NA_character_)
+    }else if(inherits(.val, "numeric") && .val == BBI_NULL_NUM && !is.na(.val)){
```

Comment:
I don't think we need one of these code paths. It looks like the `-999999999` comes in through from the json as an integer (based on the [change to the ofv test](https://github.com/metrumresearchgroup/bbr/pull/524/files#diff-ed82dffdf3c54e67fe827da182bf90b01bf71102844ef0323833bf3f572bcff7R214)), so this block doesn't ever happen. And even if it did sometimes, I don't think we could predict when it would happen in a way that makes this `NA_real_` vs `NA_integer_` distinction meaningful.

My gut is to go with `NA_real_` because it's more general. @kyleam do you have any opinion on that?

Reply:
I agree, I don't think we gain anything with the `NA_integer_` vs `NA_real_` distinction, and think it makes sense to go with `NA_real_`.

Reply:
@kyleam @seth127 I do have to have the integer and numeric for `inherits()` (will combine integer and numeric), but I can change the return value to be `NA_real_`. Will change the corresponding tests too